tensorboard/compat/proto/update.sh

```diff
@@ -49,6 +57,17 @@ find tensorboard/compat/proto/ -type f  -name '*.proto' -exec perl -pi \
   -e 's|\/\/ LINT\.|// DISABLED.|g;' \
   {} +
 
+# The tsl/ dir used to be part of TF, but now it's structured as a third_party
+# package that is "vendored" in the TF release. These two lines address the
+# before and after scenarios of that change.
+# The first line can probably be removed at some point, and the other line
+# merged with the block above.
+#
+# NOTE: Keep replacements here in sync with PROTO_REPLACEMENTS in proto_test.py.
+find tensorboard/compat/proto/ -type f  -name '*.proto' -exec perl -pi \
```

Comment:
Can these two entries just continue to be included with the original find call instead of this new one? (I couldn't tell the difference between the two).

There is this note about keeping in sync with PROTO_REPLACEMENTS but I think this change breaks that sync. PROTO_REPLACEMENTS is one list but you now have divided this into two lists here.

Reply:
Yes. I wanted to add comments, and I wasn't confident that I could add a comment there in the middle of the command, so I figured separating the command would make that clear... but I can see how that can be confusing instead, heh.

I'll merge them into the same section.